app/frontend/src/store/modules/form.js

```diff
@@ -0,0 +1,104 @@
+import formService from '@/services/formService.js';
+
+export default {
+  namespaced: true,
+  state: {
+    form: {
+      description: '',
+      id: '',
+      idps: [],
+      name: '',
+    },
+    formList: [],
+    formSubmission: {
+      confirmationId: '',
+      submission: {
+        data: {}
+      }
+    },
+    submissionList: [],
+    version: {}
```

Comment:
Right now this state module focuses on only one form. Do we have any need down the line to be able to store and "remember" multiple different forms within one session? If we do, we should perhaps consider mounting the top level state object as a guid instead; that way we could have multiple form trees stored in stead instead of just one (thinking ahead to potential vuex-persist stuff)

Reply:
Trying to keep the store as mostly just the central location for all form managing actions that a page or pages would need.
So lists of forms for the list pages, the form for the page you're on for a specific form. It can manage for now, but yeah if sections of the app need to pertain to one form it could be changed.